crates/bindings-csharp/Codegen.Tests/fixtures/diag/snapshots/Module#FFI.verified.cs

```diff
@@ -54,63 +55,37 @@ public readonly struct TestAutoIncNotInteger
                 return row;
             }
 
+            public ulong Count =>
+                SpacetimeDB.Internal.ITableView<
+                    TestAutoIncNotInteger,
+                    global::TestAutoIncNotInteger
+                >.DoCount();
+
             public IEnumerable<global::TestAutoIncNotInteger> Iter() =>
                 SpacetimeDB.Internal.ITableView<
                     TestAutoIncNotInteger,
                     global::TestAutoIncNotInteger
-                >.Iter();
+                >.DoIter();
 
             public global::TestAutoIncNotInteger Insert(global::TestAutoIncNotInteger row) =>
                 SpacetimeDB.Internal.ITableView<
                     TestAutoIncNotInteger,
                     global::TestAutoIncNotInteger
-                >.Insert(row);
-
-            public IEnumerable<global::TestAutoIncNotInteger> FilterByIdentityField(
-                string IdentityField
-            ) =>
-                SpacetimeDB
-                    .Internal.ITableView<
-                        TestAutoIncNotInteger,
-                        global::TestAutoIncNotInteger
-                    >.ColEq.Where(
-                        1,
-                        IdentityField,
-                        global::TestAutoIncNotInteger.BSATN.IdentityField
-                    )
-                    .Iter();
-
-            public global::TestAutoIncNotInteger? FindByIdentityField(string IdentityField) =>
-                FilterByIdentityField(IdentityField)
-                    .Cast<global::TestAutoIncNotInteger?>()
-                    .SingleOrDefault();
-
-            public bool DeleteByIdentityField(string IdentityField) =>
-                SpacetimeDB
-                    .Internal.ITableView<
-                        TestAutoIncNotInteger,
-                        global::TestAutoIncNotInteger
-                    >.ColEq.Where(
-                        1,
-                        IdentityField,
-                        global::TestAutoIncNotInteger.BSATN.IdentityField
-                    )
-                    .Delete();
-
-            public bool UpdateByIdentityField(
-                string IdentityField,
-                global::TestAutoIncNotInteger @this
-            ) =>
-                SpacetimeDB
-                    .Internal.ITableView<
-                        TestAutoIncNotInteger,
-                        global::TestAutoIncNotInteger
-                    >.ColEq.Where(
-                        1,
-                        IdentityField,
-                        global::TestAutoIncNotInteger.BSATN.IdentityField
-                    )
-                    .Update(@this);
+                >.DoInsert(row);
+
+            public bool Delete(global::TestAutoIncNotInteger row) =>
+                SpacetimeDB.Internal.ITableView<
+                    TestAutoIncNotInteger,
+                    global::TestAutoIncNotInteger
+                >.DoDelete(row);
+
+            public UniqueIndex<
```

Comment:
Nit for the future: we should try and generate table views in separate files.

FFI.cs should be the last resort, just for module definitions and such, where items truly need to be combined together.

Otherwise it makes caching that much worse when we need to recompute and regenerate entire FFI.cs whenever e.g. just one index in one table has changed.